README.md

```diff
@@ -670,24 +721,25 @@ make[1]: Leaving directory '/home/developer/src/test'
 developer:test > 
 ```
 
-There is no difference between the files and the test was successful. To see how `check-diff` detects unintended changes to the configuration break the `loopback-template.xml` file on purpose. Open the `loopback/templates/loopback-template.xml` file in the editor and remove the line 21:
+There is no difference between the files and the test was successful. To see how `check-diff` detects unintended changes to the configuration break the `loopback-template.xml` file on purpose. Open the `~/src/loopback/templates/loopback-template.xml` file in the editor and remove the line 21:
 ```
```

Comment:
I think using the diff in markdown can help users to see the line to remove.

```diff
           ...
             <Loopback>
               <id>{/bgp-intf}</id>
               <ipv4>
                 <address>
                   <ip>{$BGP_ADDRESS}</ip>
-                   <mask>255.255.255.255</mask>
                 </address>
               </ipv4>
             </Loopback>
           </interface>
          ...
```

Reply:
Using diff now.